sys/net/application_layer/dhcpv6/Kconfig

```diff
@@ -16,4 +16,7 @@ config DHCPV6_CLIENT_PFX_LEASE_MAX
     int "Maximum number of prefix leases to be stored"
     default 1
 
+config CONFIG_DHCPV6_CLIENT_MUD_URL
+    string "URL pointing to a Manufacturer Usage Description file"
+
 endif # KCONFIG_USEMODULE_DHCPv6
```

Comment:
```suggestion
endif # KCONFIG_USEMODULE_GNRC_DHCPV6_CLIENT_MUD_URL
endif # KCONFIG_USEMODULE_DHCPv6
```

Reply:
Thank you for your review! I added your suggested changes and included a help text (I hope it's not too long).